plugins/okta/komand_okta/tasks/monitor_logs/task.py

```diff
@@ -170,3 +178,16 @@ def get_last_collection_timestamp(self, new_logs: list, state: dict) -> str:
             new_ts = state_time
 
         return new_ts
+
+    def _get_filter_time(self, custom_config: Dict[str, int]) -> int:
+        """
+        Apply custom_config params (if provided) to the task. If a lookback value exists, it should take
+        precedence (this can allow a larger filter time), otherwise use the default value.
+        :param custom_config: dictionary passed containing `default` or `lookback` values
+        :return: filter time to be applied in `_filter_and_sort_recent_events`
```

Comment:
minor could remove this as this (`_filter_and_sort_recent_events`) doesn't exist and just mention it's used to calculate the from time?

Reply:
Also checked with a lookback of 5000 hours. It works ok.